### DISCUSSION TITLE
Youtube ad leak - possible cause?

### DISCUSSION THREAD

**Discussion by TsukiZero** [uBlockOrigin/uBlock-issues#2147] [General]
I did notice some pattern to it:
It usually happens when opening an Youtube page from another site or from a google search, but if it fails once it won't fail again for a long while.

However installing an addon that specifically forces ads to be skipped while the leaked ad runs will immediately skip said ad.

So is it possible that uBlock Origin is just not detecting the page has changed from something → Youtube?

**Comment by krystian3w** [comment]
Disable other addons with access to youtube domain.

If pre roll ads gone then other addon malform e.g. json-prune so ads is visible.
